src/markdown-pages/build-apps/permission-manage-apps.mdx

```diff
@@ -15,40 +15,31 @@ tags:
-
-User permissions vary depending on which pricing plan you are on.
-
-### New Relic One pricing plan
-
-For accounts with New Relic One pricing, there are permissions differences for basic users and full users:
+There are several factors affecting your New Relic One app permissions. 
 
-**Full users** have the Nerdpack manager role and have full capabilities for creating and managing New Relic One applications, as well as accessing all types of applications in the New Relic One catalog.
+## Basic user restrictions 
 
-A **basic user** can develop and view their own local New Relic One apps, but they ***cannot***:
+The most important permissions factor is [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#user-type): 
 
-- Subscribe other users to apps they’ve created.
-- Access or manage apps in the New Relic One catalog.
-- Access apps in the entity explorer sidebar.
+* **Basic users** can only develop New Relic One apps locally. They can’t publish or subscribe to an app, whether it's deployed in their account, available in [the public catalog](https://opensource.newrelic.com/nerdpacks/), or one they've built. 
+* **Full users** can theoretically do everything, but there may be restrictions due to custom role assignments (see below). 
 
-### Original product-based pricing
+## Role-related permissions 
 
-For accounts on our original product-based pricing, here are access details:
+Full users have role-related rules that impact their ability to subscribe accounts to New Relic One apps. How this works depends on your account/user model: 
 
-**Subscribe to publicly available applications**
+* [Original user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models): Owners and Admins can manage New Relic One apps, as can users specifically assigned the **Nerdpack manager** add-on role. For more details about how account access works for users, see [Account access](#account-access)
+* [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models): the ability to publish and subscribe to apps is dependent on the "modify Nerdpacks" [capability](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#roles). The **All product admin** role has this capability, as do the default groups [**Admin** and **User**](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#groups). 
```

Comment:
Can't an account manager can also create a custom group/role that has this privilege?